packages/server/src/fhir/operations/agentpush.ts

```diff
@@ -114,3 +121,17 @@ async function pushToAgent(req: Request): Promise<[OperationOutcome] | [Operatio
   // 1. The agent will respond with a message on the channel
   // 2. The timer will expire and the request will timeout
 }
+
+function validateParams(params: AgentPushParameters): void {
```

Comment:
Mental note, as discussed offline, in the future these checks should be handled by `parseParameters()` for required fields.